text_generation/causal_lm/cpp/CMakeLists.txt

```diff
@@ -4,25 +4,30 @@
 cmake_minimum_required(VERSION 3.15)
 project(causal_lm)
 
-add_subdirectory(../../../thirdparty/openvino_tokenizers/ "${CMAKE_CURRENT_BINARY_DIR}/openvino_tokenizers/")
+if(TARGET openvino_tokenizers)
+    set(OPENVINO_TOKENIZERS_PATH $<TARGET_FILE:openvino_tokenizers>)
+else()
+    set(OPENVINO_TOKENIZERS_PATH ${CMAKE_CURRENT_SOURCE_DIR}/../../bin/openvino_tokenizers.dll)  # TODO: I'll go away after the generate() gets a way to find openvino_tokenizers
```

Comment:
@Wovchena 
can you help to drop it?
we agreed to copy tokenizers to folder with `openvino_genai.dll` so it can find tokenizers by `abs_path(openvino_genai.dll) / "openvino_tokenizers.dll"`